### ISSUE TITLE
Human eye MRI analysis

### ISSUE THREAD

**Issue by skill51988** [bowang-lab/MedSAMSlicer#10]
Hi,

I am currently working on characterising the human eye using T1 MRI images.
It seems to successfully segment the eye in the axial plane, yet the outcome in the z-axis is not ideal 

I wonder whether this plug-in is trainable and if you have any suggestions.

Thanks!!

**Comment by JunMa11** [comment]
Hi @skill51988 ,

Thanks for your interest very much.

The low performance in the z-axis is expected since the training data didn't contain data in z-axis.

Nevertheless, the model can be easily fine-tuned to the new data. here is an example. 
https://github.com/bowang-lab/MedSAM/blob/LiteMedSAM/README.md
